### PR TITLE
Added some directory info extension unit tests.

### PR DESCRIPTION
**Pull request by christopherbauer**
Please verify that the setup methods do not cause horror and/or dismay

### CODE REVIEW

**Comment by JimBobSquarePants** [comment]
Cheers. Will do! :smile:

**Comment by christopherbauer** [comment]
Wasn't sure if this would be here in the morning or merged, but I figure I'd explain just a bit;

Didn't pull the setup logic out to a separate function because I'm pretty strict about "if you need it 3x refactor, else keep it separate" mentality. 
These operations are setup/teardowns so the tests are repeatable, and if directory info is extended for other things such as moving a directory I don't want those tests to impact one-another.

File: src/ImageProcessor.Web.UnitTests/Extensions/DirectoryInfoExtensionsUnitTests.cs [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            /// </summary>
+            private IEnumerable<string> directoryList;
+
+            /// <summary>
+            /// The setup directories.
+            /// </summary>
+            [SetUp]
+            public void SetupDirectories()
+            {
+                this.directoryList = Enumerable.Range(1, this.directoryCount).Select(i => $"{TestDirectoryRoot}/TestDirectory{i}");
```
Comment (JimBobSquarePants) [inline src/ImageProcessor.Web.UnitTests/Extensions/DirectoryInfoExtensionsUnitTests.cs]: Didn't know you could do this. Looks like the new C#6 string templating.

Comment (christopherbauer) [inline src/ImageProcessor.Web.UnitTests/Extensions/DirectoryInfoExtensionsUnitTests.cs]: Woops it is, I didn't even think about it. We probably shouldn't be doing that yet huh?

Comment (JimBobSquarePants) [inline src/ImageProcessor.Web.UnitTests/Extensions/DirectoryInfoExtensionsUnitTests.cs]: :smile: ....I thought about switching over but decided it was too risky for anyone collaborating. I'll need to check my AppVeyor settings as that didn't pick up on the issue.

**Comment by christopherbauer** [comment]
I rolled back the string formatting to be compliant to .net 4. String interpolation was fun while it lasted! Do you prefer squash or just rolling the whole thing in as-is?

**Comment by JimBobSquarePants** [comment]
I'm gonna merge the whole thing as is. We can always refactor tests after.

**Pull request merged**